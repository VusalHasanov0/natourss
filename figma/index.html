<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="css/bootstrap.min.css" >
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/fontawesome.min.css" integrity="sha512-cHxvm20nkjOUySu7jdwiUxgGy11vuVPE9YeK89geLMLMMEOcKFyS2i+8wo0FOwyQO/bL8Bvq1KMsqK4bbOsPnA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
	<link rel="stylesheet" type="text/css" href="style.css">
	<title>Figma Tasarımı</title>
</head>
<body>

	<section class="stage1">
		<div class="container">
			<div class="image">
				<img src="img/foto2.png" class="logo1 mt-4 mb-3 ">
				<button class="btn btn-light button  "><i class="fa fa-search "></i></button>
			</div>
			<nav class="navbar navbar-expand-sm web-menu justify-content-center">
				
				
					<ul class="navbar-nav nav-pills">


						<li class="nav-item mr-2"><a href="#" class="nav-link">hom</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Corporate</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Portfolio</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Corporate Responsibility</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Reports</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Press Room</a></li>
						<li class="nav-item mr-2"><a href="careers-details.html" class="nav-link"> Careers</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link"> Contact</a></li>
					</ul>
			</nav>

			<nav class="navbar navbar-expand-sm mobile-menu justify-content-center">
				<button class="navbar-toggler  " type="button" data-toggle="collapse" data-target="#navbarNav">
					<span class="menu-logo" >MENU</span>
				</button>
				<div class=" collapse navbar-collapse justify-content-center navbar-expand-sm" id="navbarNav">
					<ul class="navbar-nav nav-pills">


						<li class="nav-item mr-2"><a href="#" class="nav-link">Home</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Corporate</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Portfolio</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Corporate Responsibility</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Reports</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link">Press Room</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link"> Careers</a></li>
						<li class="nav-item mr-2"><a href="#" class="nav-link"> Contact</a></li>
					</ul>
				</div>

			</nav>

			<div class="stage1-plan1">
				<div class="row">
					<div class="col-sm-6">

						<h1 class="mb-4">A Secret Mediterranean jewel is now coming to life</h1>

						<p class="mb-4">Portonovi Montenegro will be a destination unmatched in the Mediterranean</p>

						<button class="btn explore">Explore  <i class="fa-sharp fa-solid fa-right-long ml-3"></i></button>

						<button class="scroll mt-2">SCROLL DOWN <i class="fa-solid fa-arrow-right ml-2"></i></button>
					</div>
					<div class="col-sm-6 ">
						<div class="buttons justify-content-end">
							<button class="mb-2"><i class="fa-brands fa-twitter fa-rotate-270"></i></button>
							<button class=" mb-2"><i class="fa-brands fa-linkedin-in fa-rotate-270"></i></button>
							<button class=" mb-2"><i class="fa-brands fa-instagram fa-rotate-270"></i></button>
							<button class=" mb-2"><i class="fa-brands fa-facebook-f fa-rotate-270"></i></button>


							<button class="scroll1 mt-2">Follow Us <i class="fa-solid fa-arrow-right ml-2"></i></button>
						</div>
					</div>
				</div>
			</div>
		</div>
	</section>


	<script src="js/jquery-3.6.3.min.js" ></script>
	<script src="js/popper.js" ></script>
	<script src="js/bootstrap.min.js" ></script>
	<script  src="main.js"></script>

</body>
</html>