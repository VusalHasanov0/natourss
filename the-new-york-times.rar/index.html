<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>The New York Times - Breaking News, US News, World News and Videos</title>
    <link rel="stylesheet" href="styles.css">
</head>
<body>
    <header class="header" >
        <section class="header_item">
           
            <div class="buttons">
                <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
                <a href="#"><i class="fa fa-search" aria-hidden="true"></i></a>
            </div>

            <div class="language">
                <ul class="language-ul">
                    <li><a href="#">U.S.</a></li>
                    <li><a href="#">INTERNATIONAL</a></li>
                    <li><a href="#">CANADA</a></li>
                    <li><a href="#">ESPAÑOL</a></li>
                    <li><a href="#">中文</a></li>
                </ul>
               
            </div>
           
            <div class="button2">
                <button><a href="#" class="button2_item">SUBSCRIBE FOR $0.25/WEEK</a></button>
            </div>
            <div class="button3">
                <button><a href="#" class="button3_item">LOG IN</a></button>
            </div>
            
            <div class="calendar">
                <h2>Monday, February 27, 2023</h2>
                <h3><a href="#">Today’s Paper</a></h3>
            </div>
            <div class="image">
                <a href="#">
                    <img src="images.png" alt="">
                </a>
            </div>
            <div class="categories">
                <ul class="list">
                    <li class="list-item">
                        <a href="#">World</a>
                    </li>
                    <li class="list-item">
                        <a href="#">U.S.</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Politics</a>
                    </li>
                    <li class="list-item">
                        <a href="#">N.Y.</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Business</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Opinion</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Science</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Health</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Sports</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Arts</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Books</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Style</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Food</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Travel</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Magazine</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Real Estate</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Cooking</a>
                    </li>
                    <li class="list-item">
                        <a href="#">The Athletic</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Wirecutter</a>
                    </li>
                    <li class="list-item">
                        <a href="#">Games</a>
                    </li>
                 
                </ul>
            </div>
            <div class="categories1" aria-hidden="true" style="visibility: hidden; transform: translateY(-70px);">
                <div class="categories2">
                    <ul class="list">
                        <li class="list-item">
                            <a href="#">World</a>
                        </li>
                        <li class="list-item">
                            <a href="#">U.S.</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Politics</a>
                        </li>
                        <li class="list-item">
                            <a href="#">N.Y.</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Business</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Opinion</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Science</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Health</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Sports</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Arts</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Books</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Style</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Food</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Travel</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Magazine</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Real Estate</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Cooking</a>
                        </li>
                        <li class="list-item">
                            <a href="#">The Athletic</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Wirecutter</a>
                        </li>
                        <li class="list-item">
                            <a href="#">Games</a>
                        </li>
                     
                    </ul>
                </div>

            </div>
            <div class="weather">
                <div class="weather-item">
                    <div class="weather-icon1">
                        <span>4°C</span>
                    </div>
                    <div class="weather-icon2">
                        <span>16°</span>
                    </div>
                    <div class="weather-icon3">
                        <span>5°</span>
                    </div>
                    <div>
                        <i class="fa-sharp fa-regular fa-cloud-fog"></i>
                    </div>
                    

                
                


            </div>
            </div>
    </section>
    </header>
    <div class="main">
        <div class="container">
        <div class="container1">
        <div class="container2">
        <p class="main-1"><a href="#">Supreme Court's Majority </a> </p>
        <p class="main-2"><a href="#">Appears Skeptials of Student </a></p>
        <p class="main-3"><a href="#">Loan Forgiveness Plan</a></p>
    </div>
        <div class="plan">
            <ul class="plan-ul">
                <li class="ul-li1"><a href="#"> The Supreme Court is hearing arguments over the legality of President Biden’s plan to wipe out more than $400 billion in student debt.</a></li>
                <li class="ul-li2"> <a href="#"> Chief Justice John Roberts indicated that the administration had violated separation-of-powers principles by acting without Congress’s authorization on the plan.</a></li>
            </ul>
            <p class="update">
                <a href="#">See More Updates
                 <span class="update-1">
                     9
                     +
                 </span>
             </a> 
             </p>
        </div>
       
    
        <div class="plan-2">
            <span class="plan-2-item"><a href="#">Here’s what to know about President Biden’s student loan forgiveness plan.
            </a></span>
            <div class="plan-2-item1">
                <a href="#">14 MIN READ</a>
            </div>
        </div>
    </div>
        
        <div class="photo">
                <a href="#"><img src="28dc-scotus-cmbz-threeByTwoMediumAt2X.webp" alt=""></a>
                <div class="photo-word">
                    <p class="word1"><a href="#">Senator Elizabeth Warren speaks during a rally for student debt relief.</a></p>
                    <span class="word2"> <a href="#">Anna Rose Layden for The New York Times</a> 
                    </span>
                </div>
        </div>
        
    </div>
    <div class="container3">
        <div class="photo1">
           <a href="#"> <img src="27SAGQQ-swaps-02-tpkc-threeByTwoMediumAt2X.webp" alt=""></a>
           <div class="photo-word1">
            <span class="word-1"> <a href="#">Jordan Strauss/Invision, via Associated Press
            </a></span>
        </div>
        </div>
        <div class="plan-3-item1">
            <a href="#"> 
                Awards Without Gender Categories? Celebrities Debate.
            </a>
        </div>
        <div class="plan-3-item2">
            <a href="#">
                Nominees at the Screen Actors Guild Awards in Los Angeles on Sunday were split on combining award show categories for best actor and best actress.
                </a>
        </div>
        <div class="plan-3-item3">
            <a href="#">
                5 MIN READ
    
            </a>
        </div>
        </div>
        
        
    </div>
    </div>

    
</body>
</html>