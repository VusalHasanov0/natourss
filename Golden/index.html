<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Golden Website</title>
    <link rel="stylesheet" href="css/bootstrap.min.css" >
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/fontawesome.min.css" integrity="sha512-cHxvm20nkjOUySu7jdwiUxgGy11vuVPE9YeK89geLMLMMEOcKFyS2i+8wo0FOwyQO/bL8Bvq1KMsqK4bbOsPnA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link rel="stylesheet" type="text/css" href="style.css">
    <link rel="stylesheet" type="text/css" href="css/animate.css">

</head>
<body>

    <section class="stage1">
        <div class="container ">
            <div class="row">
                <nav class="navbar   navbar-dark fixed-top navbar-expand-md ">
                    <button class="navbar-toggler navbar-toggler-right " type="button" data-toggle="collapse" data-target="#navbarNav">
                        <span class="  navbar-toggler-icon "></span>
                    </button>
                    <a href="#" class="navbar-brand"><img width="75px" class="  logo navbar-brand" src="img/logo.png"></a>


                    <div class="collapse navbar-collapse navbar-expand-md justify-content-end" id="navbarNav">
                        <ul class="navbar-nav nav-pills ">
                            <li class="nav-item"><a href="#" class="nav-link ">Home</a></li>
                            <li class="nav-item"><a href="#stage2" class="nav-link ">Services</a></li> 
                            <li class="nav-item"><a href="#stage3" class="nav-link ">Portfolio</a></li>
                            <li class="nav-item"><a href="#stage4" class="nav-link ">About</a></li>
                            <li class="nav-item"><a href="#stage7" class="nav-link ">Contact</a></li>
                        </ul>
                    </div>
                </nav>
            </div>
            <div class="row text-center">
                <div class="col-sm-12">
                  <h1 class="stage1-h1 lead">Welcome To BMW</h1>
              </div>
              <div class="col-sm-12">
                  <h2 class="stage1-h2">Its Nice To Meet You</h2>
              </div>
              <div class="col-sm-12 mt-4">
                <a href="#"><button type="button" class="btn btn-primary btn-lg stage1-button" >TELL ME MORE</button></a>
            </div>
        </div>
    </div>
</section>

<section class="stage2" id="stage2">
  <div class="container">
      <div class="stage2-plan1 text-center wow fadeInUp" data-wow-duration="2s">
          <h3>SERVICES</h3>
          <p>Proin iaculis purus consequat sem cure</p>
      </div>
      <div class="stage2-plan2">
          <div class="row">
              <div class="col-sm-4 text-center wow fadeInLeft" data-wow-duration="2s">
                  <div class="icon">
                      <i class="fa-solid fa-car fa-3x  " style="color: #0066b3;"></i>
                  </div>
                  <h4>E-Commerce</h4>
                  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
                      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
                  </p>
              </div>
              <div class="col-sm-4 text-center wow fadeInUp" data-wow-duration="2s">
                  <div class="icon">
                      <i class="fa-solid fa-desktop fa-3x  " style="color: #0066b3;"></i>
                  </div>
                  <h4>Responsive Web</h4>
                  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
                      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
                  </p>
              </div>
              <div class="col-sm-4 text-center wow fadeInRight" data-wow-duration="2s">
                  <div class="icon">
                      <i class="fa-solid fa-lock fa-3x  " style="color: #0066b3;"></i>
                  </div>
                  <h4>Web Security</h4>
                  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
                      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
                  </p>
              </div>
          </div>
      </div>
  </div>
</section>

<section class="stage3" id="stage3">
  <div class="container">
      <div class="stage3-plan1 text-center">
        <h3>OUR DESIGN</h3>
        <p>Proin iaculis purus consequat sem cure</p>
    </div>
    <div class="stage3-plan2">
      <div class="row stage3-plan2-row">
          <div class="col-sm-4 mb-4">
              <div class="card">
                  <img class="card-img-top" src="img/card1.jpg">
                  <div class="card-body stage3-plan1-card-body ">
                      <h4 class="card-title"> BMW M4 F82 </h4>
                      <p class="card-text text-muted">Branding</p>
                  </div>
              </div>
          </div>
          <div class="col-sm-4 mb-4">
              <div class="card">
                  <img class="card-img-top" src="img/card2.jpg">
                  <div class="card-body stage3-plan1-card-body">
                      <h4 class="card-title"> BMW M3 E46 </h4>
                      <p class="card-text text-muted">Branding</p>
                  </div>
              </div>
          </div>
          <div class="col-sm-4 ">
              <div class="card">
                  <img class="card-img-top" src="img/card3.jpg">
                  <div class="card-body stage3-plan1-card-body">
                      <h4 class="card-title"> BMW M3 E36 </h4>
                      <p class="card-text text-muted">Branding</p>                  </div>
                  </div>
              </div>
          </div>

          <div class="row">
              <div class="col-sm-4 mb-4">
                  <div class="card">
                    <img class="card-img-top" src="img/card4.jpg">
                    <div class="card-body stage3-plan1-card-body">
                      <h4 class="card-title"> BMW M3 E30 </h4>
                      <p class="card-text text-muted">Branding</p>
                  </div>
              </div>
          </div>
          <div class="col-sm-4 mb-4">
              <div class="card">
                  <img class="card-img-top" src="img/card5.jpg">
                  <div class="card-body stage3-plan1-card-body">
                      <h4 class="card-title "> BMW 1974 OLD </h4>
                      <p class="card-text text-muted">Branding</p>
                  </div>
              </div>
          </div>
          <div class="col-sm-4 mb-4">
              <div class="card">
                  <img class="card-img-top" src="img/card6.jpeg">
                  <div class="card-body stage3-plan1-card-body">
                      <h4 class="card-title"> BMW M5 M50I </h4>
                      <p class="card-text text-muted">Branding</p>
                  </div>
              </div>
          </div>
      </div>
  </div>
</div>
</section>


<section class="stage4" id="stage4">
    <div class="container">
        <div class="stage4-plan1 text-center">
            <h3>ABOUT US</h3>
            <p>Proin iaculis purus consequat sem cure</p>
        </div>
        <div class="stage4-plan2 text-center">
            <div class="row">

                <div class="col-sm-4 ">
                    <img class="stage4-logo1" src="img/stage4-logo.jpg">
                </div>
                <div class="col-sm-4  text-right justify-content-end">
                    <h4>JULY 2010</h4>
                    <h4>Our humble Beginnings</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
                        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
                        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
                    consequat.</p>
                </div>

            </div>
            <div class="row">
                <div class="col-sm-4"></div>
                <div class="col-sm-4">
                    <img class="stage4-logo1" src="img/stage4-logo2.jpg">
                </div>
                <div class="col-sm-4 text-left">
                    <h4>JULY 2010</h4>
                    <h4>Our humble Beginnings</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
                        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
                        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
                    consequat.</p>
                </div>             
            </div>

            <div class="row">
                <div class="col-sm-4">
                    <img class="stage4-logo1" src="img/stage4-logo3.jpg">
                </div>
                <div class="col-sm-4 text-right">
                    <h4>JULY 2010</h4>
                    <h4>Our humble Beginnings</h4>
                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
                        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
                        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
                    consequat.</p>
                </div>

            </div>

            <div class="row">
                <div class="col-sm-4 offset-sm-4">
                    <div class="stage4-about">
                        <p class="about-text">OUR <br> 
                            STORY <br>
                            CONTINUES <br>
                        ...</p>
                    </div>
                    
                </div>
            </div>

        </div>
    </div>
</section>

<section class="stage5">
    <div class="container">
        <div class="stage5-plan1 text-center">
            <h3>OUR AMAZING TEAM</h3>
            <p>Proin iaculis purus consequat sem cure</p>
        </div>
        <div class="stage5-plan2">
            <div class="row">
                <div class="col-sm-4 text-center">
                    <div class="stage5-logo"></div>
                    <h4>Kimberly Thompson</h4>
                    <h5>Marketer</h5>
                    <ul class="stage5-icons">
                        <li class="stage5-icon"><i class="fa-brands fa-twitter"></i></li>
                        <li class="stage5-icon"><i class="fa-brands fa-facebook"></i></li>
                        <li class="stage5-icon"><i class="fa-brands fa-pinterest"></i></li>
                        
                    </ul>
                </div>
                <div class="col-sm-4 text-center">
                    <div class="stage5-logo"></div>
                    <h4>Rico Massimo</h4>
                    <h5>Coder</h5>
                    <ul class="stage5-icons">
                        <li class="stage5-icon"><i class="fa-brands fa-twitter"></i></li>
                        <li class="stage5-icon"><i class="fa-brands fa-facebook"></i></li>
                        <li class="stage5-icon"><i class="fa-brands fa-pinterest"></i></li>
                        
                    </ul>
                </div>
                <div class="col-sm-4 text-center">
                    <div class="stage5-logo"></div>
                    <h4>Uku Mason</h4>
                    <h5>Graphic Designer</h5>
                    <ul class="stage5-icons">

                        <li class="stage5-icon"><i class="fa-brands fa-facebook"></i></li>
                        <li class="stage5-icon"><i class="fa-brands fa-pinterest"></i></li>
                        
                    </ul>
                </div>

            </div>

            <div class="row text-center">
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt <br>
                ut labore et dolore magna aliqua. Ut enim ad minim veniam</p>
            </div>
        </div>
    </div>
</section>

<section class="stage6 ">
    <div class="container">
        <div class="row text-center">
            <div class="col-sm-3">
                <img width="200px" class="stage6-envato" src="img/stage6-envato.png">
            </div>
            <div class="col-sm-3">
                <img  width="200px" src="img/stage6-wordpress.png">
            </div>
            <div class="col-sm-3">
                <img width="200px" src="img/stage6-aetuts.png">
            </div>
            <div class="col-sm-3">
                <img width="200px" src="img/stage6-microlancer.png">
            </div>
        </div>
    </div>
</section>


<section class="stage7" id="stage7">
    <div class="container">
        <div class="stage7-plan1 text-center">
            <h3>CONTACT US</h3>
            <p>Proin iaculis purus consequat sem cure</p>
        </div>
        <div class="stage7-plan2">
            <form class="stage7-form" role="form" method="post">
                <div class="row">
                    <div class="col-sm-6">
                        <div class="stage7-form-elements">
                            <input type="text" class="stage7-form-element" name="name" placeholder="Your Name">
                        </div>
                        <div class="stage7-form-elements">
                            <input type="email" class="stage7-form-element"  name="email" placeholder="Your Email">
                        </div>
                        <div class="stage7-form-elements">
                            <input type="text" class="stage7-form-element" name="subject" placeholder="Subject">
                        </div>
                    </div>
                    <div class="col-sm-6">
                        <div class="stage7-form-elements">
                            <textarea class="stage7-form-element stage7-form-textarea" name="message" placeholder="Your Message"></textarea>
                        </div>
                    </div>
                </div>
                <div class="row justify-content-center">
                    <div class="stage7-form-elements" >
                        <input type="submit" name="submit" class="btn btn-primary btn-lg stage1-button" value="SEND MESSAGE" >
                    </div>
                </div>
            </form>
        </div>
    </div>
</section>


<footer class="footer">
    <div class="container">
        <div class="row">
            <div class="col-sm-4">
                <p> &copy; Copyright 2023 Bmw.com</p>
            </div>
            <div class="col-sm-4">
                <ul class="footer-social-icons text-center">
                    <li class="footer-social-icon">
                       <i class="fa-brands fa-twitter"></i>
                   </li>
                   <li class="footer-social-icon">
                    <i class="fa-brands fa-facebook"></i>
                </li>
                <li class="footer-social-icon">
                   <i class="fa-brands fa-pinterest"></i>
               </li>
               <li class="footer-social-icon">
                   <i class="fa-brands fa-google-plus"></i>
               </li>
           </ul>
       </div>
   </div>
</div>
</footer>


















<script src="js/jquery-3.6.3.min.js" ></script>
<script src="js/popper.js" ></script>
<script src="js/bootstrap.min.js" ></script>
<script  src="main.js"></script>
<script src="js/wow.js"></script>
<script >
    new WOW().init();
</script> 
</body>


</html>